config/i18n.js

```diff
@@ -1,4 +1,5 @@
 export default {
   langPickerRender: true,
-  defaultLang: ''
+  langList: ['en-US', 'ru-RU'],
+  fallbackLng: 'en-US'
```

Comment:
Changing it to `ru-RU` does not have any effect

Reply:
@mairh We are matching the list of preferred languages in order of their priority specified in web browser by the user to the list of languages available in our system. Only if that fails we choose `fallbackLng`.

Reply:
I see. Since I didn't work on this feature, I wasn't aware that we are detecting user's browser lang pref. I was under the impression that we need to set the default language in the config.